week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,21 +20,41 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
+  fillUp: function () {
+    return this.volume === 100;

```

Comment:
You only use '===' as a comparison operator, that is, in 'if' expressions. Use a single '=' for assigning values.